public/dorequest.php

```diff
@@ -26,6 +28,16 @@
 use Illuminate\Http\JsonResponse;
 use Illuminate\Support\Carbon;
 
+$requestType = request()->input('r');
+$handler = match ($requestType) {
+    'codenotes2' => new GetCodeNotes(),
+    'submitcodenote' => new SubmitCodeNote(),
+    default => null,
+};
+if ($handler) {
+    return $handler->execute(request());
+}
```

Comment:
Once everything is converted, default will return an error handler, and this file would end here.